tests/update-ci-snapshots/Dockerfile

```diff
@@ -69,4 +69,4 @@ RUN SKIP_POSTINSTALL=1 npm ci
 COPY . .
 
 # Now build the app. We only need the code-studio built for e2e tests.
-RUN VITE_CORE_API_URL=http://host.docker.internal:10000/jsapi npm run build:app
\ No newline at end of file
+RUN npm run build:app
```

Comment:
When we were using CRA, there was no proxy server, so if we wanted to point to a JS API not reachable at `/jsapi` on our preview server, we needed to build it w/ a URL instead.

Prior to #1102 we only proxied requests in dev mode for Vite. With that change, we proxy requests for the Vite preview server as well. So now we can build expecting `/jsapi` to resolve properly since it will go through the proxy which defaults to `localhost:10000`